src/components/ChallengeEditor/ChallengeView/index.js

```diff
@@ -163,6 +163,15 @@ const ChallengeView = ({
                 readOnly
               />
             )}
+            <div>
+              { challenge.discussions && challenge.discussions.map(d => (
+                <div className={cn(styles.row, styles.topRow)}>
```

Comment:
Each rendered element in the list should have a unique `key` prop (for example `d.id` can be used):

![image](https://user-images.githubusercontent.com/146016/98414553-42989580-2084-11eb-8c75-d2070ec7e291.png)

Reply:
Fixed! Thanks!